nucliadb_vectors/src/data_point/node.rs

```diff
@@ -149,19 +180,23 @@ mod tests {
     }
     const NO_LABELS: [&[u8]; 0] = [];
     const LABELS: [&[u8]; 3] = [b"L1", b"L2", b"L3"];
+    const NO_METADATA: Option<&[u8]> = None;
```

Comment:
A variable named `NO_METADATA` makes me think about something *without* metadata (no metadata). Not sure about what it really means.

Reply:
good, is supposed to make you think about that. Is for testing, I use it for the vectors I want to index in the text without metadata.